src/main/scala/com.snowplowanalytics.sauna/observers/Observer.scala

```diff
@@ -86,6 +89,23 @@ object Observer {
     }
   }
 
+
+
+  /**
+   * Record has been received from AWS Kinesis Stream
+   *
+   * @param streamName name of kinesis stream
+   * @param record kinesis record
```

Comment:
The `record` parameter isn't used in the argument list, and some of the function's arguments are omitted from the comment. Can you change this scaladoc to match the following template?

```scala
  /**
   * Record has been received from AWS Kinesis Stream
    * 
   * @param streamName name of kinesis stream
   * @param seqNr ???
   * @param data ???
   * @param observer ???
   */
```